docs/configuration/cluster_manager/cds.rst

```diff
@@ -0,0 +1,64 @@
+.. _config_cluster_manager_cds:
+
+Cluster discovery service
+=========================
+
+The cluster discovery service (CDS) is an optional API that Envoy will call to dynamically fetch
+cluster manager members. Envoy will reconcile the API response and add, modify, or remove known
+clusters depending on what is required.
+
+.. code-block:: json
+
+  {
+    "cluster": "{...}",
```

Comment:
No problem - its not an issue for me, was just thinking about the less familiar reader :-)